.neutrinorc.js

```diff
@@ -26,13 +28,13 @@ module.exports = {
         env: {
           HOST: DEFAULT_HOST,
           PORT: DEFAULT_PORT,
-          BALROG_ROOT_URL: 'https://localhost:8010',
-          AUTH0_CLIENT_ID: 'GlZhJQfx52b7MLQ19AjuTJHieiB4oh1j',
-          AUTH0_DOMAIN: 'balrog-localdev.auth0.com',
-          AUTH0_AUDIENCE: 'balrog-localdev',
-          AUTH0_RESPONSE_TYPE: 'token id_token',
-          AUTH0_SCOPE: 'full-user-credentials openid profile email',
-          AUTH0_REDIRECT_URI: `http://localhost:${port}/login`,
+          BALROG_ROOT_URL: process.env.BALROG_ROOT_URL || 'https://localhost:8010',
+          AUTH0_CLIENT_ID: process.env.AUTH0_CLIENT_ID || 'GlZhJQfx52b7MLQ19AjuTJHieiB4oh1j',
+          AUTH0_DOMAIN: process.env.AUTH0_DOMAIN || 'balrog-localdev.auth0.com',
+          AUTH0_AUDIENCE: process.env.AUTH0_AUDIENCE || 'balrog-localdev',
+          AUTH0_RESPONSE_TYPE: process.env.AUTH0_RESPONSE_TYPE || 'token id_token',
+          AUTH0_SCOPE: process.env.AUTH0_SCOPE || 'full-user-credentials openid profile email',
+          AUTH0_REDIRECT_URI: process.env.AUTH0_REDIRECT_URI || `http://localhost:${port}/login`,
```

Comment:
The values specified here are the default ones. Once `dotenv` injects variables before this file runs, all of these values will be ignored if they are defined in the `.env` file. That being said, I think we can remove `process.env.[..]` from here.

Reply:
Isn't this what we want? For local development, what's in the repo is fine (we'll use the defaults specified above). When building for stage or prod, we'll drop in a `.env`, which will override them.

Reply:
By the time `.neutrinorc.js` runs, the environment variables would have already been set. Neutrino will ignore all of the key value pairs that are already defined.

Reply:
You could technically just have:

```js
{
  env: [
    'AUTH0_AUDIENCE',
    'AUTH0_RESPONSE_TYPE',
    // ...
  ],
};
```

Neutrino will throw a warning if it notices that any of these values are not already defined in the environment. https://master.neutrinojs.org/packages/web/#environment-variables

Reply:
Ah, I didn't realize Neutrino would automatically look in the environment for overrides...that makes things simpler!

Reply:
Given that, it looks like this file doesn't need to change at all...

Reply:
Indeed :)